src/dotnet/ToolPackageObtainer/ToolPackageObtainer.cs

```diff
@@ -89,7 +90,20 @@ IProjectRestorer projectRestorer
                     packageId,
                     packageVersion,
                     "tools",
-                    targetframework));
+                    targetframework,
+                    "any"));
+        }
+
+        private static void MoveToConcreteWithOverride(
+            DirectoryPath concreteVersionIndividualToolVersion,
+            DirectoryPath individualToolVersion)
```

Comment:
This also seems like a weird name. Is this the temporaryToolDirectory?

Reply:
fixed, change to versioned vs temporary. Directory is not needed, since the type is directory already